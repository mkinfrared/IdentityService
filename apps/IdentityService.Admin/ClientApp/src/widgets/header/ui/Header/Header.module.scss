.Header {
  background-color: palegreen;
  height: 50px;
  position: static;
  width: 100%;
}
