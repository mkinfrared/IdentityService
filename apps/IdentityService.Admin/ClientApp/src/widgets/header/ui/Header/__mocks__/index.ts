export { default } from "./Header.mock";
