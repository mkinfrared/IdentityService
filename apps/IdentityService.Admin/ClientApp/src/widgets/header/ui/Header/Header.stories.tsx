import { Meta, Story } from "@storybook/react";

import Header from "./Header";
import { HeaderProps } from "./Header.type";
import css from "./Story.module.scss";

export default {
  title: "COMPONENTS/Header",
  component: Header,
  parameters: {
    componentSubtitle: "Subtitle goes here",
  },
} as Meta;

const Template: Story<HeaderProps> = (args) => (
  <div className={css.Story}>
    <Header {...args} />
  </div>
);

const Default = Template.bind({});

Default.args = {};

Default.parameters = {
  docs: {
    storyDescription: "Story description",
  },
};

export { Default };
