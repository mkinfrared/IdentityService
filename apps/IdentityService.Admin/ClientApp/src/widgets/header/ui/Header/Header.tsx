import { classNames } from "@identity-service/core";

import css from "./Header.module.scss";
import { HeaderProps } from "./Header.type";

const Header = ({ className }: HeaderProps) => (
  <div className={classNames(css.Header, className)} data-testid="Header">
    Header works!!
  </div>
);

export default Header;
