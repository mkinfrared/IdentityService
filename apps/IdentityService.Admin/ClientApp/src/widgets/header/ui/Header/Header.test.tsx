import { render } from "@testing-library/react";

import Header from "./Header";

describe("<Header />", () => {
  const Component = <Header />;

  it("should be defined", () => {
    expect(Header).toBeDefined();
  });

  it("should match the snapshot", () => {
    const { container } = render(Component);

    expect(container).toMatchSnapshot();
  });

  it("should contain a data test id", () => {
    const { getByTestId } = render(Component);
    const element = getByTestId("Header");

    expect(element).toBeDefined();
  });
});
