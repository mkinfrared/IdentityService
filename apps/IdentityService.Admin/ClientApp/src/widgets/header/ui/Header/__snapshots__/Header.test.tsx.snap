// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match the snapshot 1`] = `
<div>
  <div
    class="Header"
    data-testid="Header"
  >
    Header works!!
  </div>
</div>
`;
